TrustModel_Personas.md

```diff
@@ -0,0 +1,200 @@
+ are the only code that can operate over the data. However any vulnerabilities within this code 
+ are not mitigated by using confidential containers, the Cloud Native Security Whitepaper 
+ details Lifecycle aspects that relate to the security of the code being placed into containers 
+ such as Static/Dynamic Analysis, Security Tests, Code Review etc which must still be followed. 
+
+![Personas Model](./images/PersonaModel.svg)
+
+Any of these personas could attempt to perform malicious actions:
+
+### Infrastructure Operator
+This persona has privileges within the Cloud Infrastructure which includes the hardware and 
+firmware used to provide compute, network and storage to the Cloud Native solution.
+They are responsible for availability of infrastructure used by the cloud native environment.
+- Have access to the physical hardware.
+- Have access to the processes involved in the deployment of compute/storage/memory used by any 
+  orchestration components and by the workload.
+- Have control over TEE hardware availability/type.
+- Responsibility for applying firmware updates to infrastructure including the TEE Technology.
+
+Example : Cloud Service Provider (CSP), Infrastructure as a Service (IaaS), Site Reliability Engineer, etc.
+(SRE)
+
+### Orchestration Operator
+This persona has privileges within the Orchestration/Cluster.
+They are responsible for deploying a solution into a particular cloud native environment and 
+managing the orchestration environment.
+For managed cluster this would also include the administration of the cluster control plane.
+- Control availability of service.
+- Control webhooks and deployment of workloads.
+- Control availability of cluster resources (data/networking/storage) and cluster
+  services (Logging/Monitoring/Load Balancing) for the workloads.
+- Control the deployment of runtime artifacts initially required by the TEE during 
+  initialisation. 
+ These boot images once initialised will receive the workload.
+
+Example : A kubernetes administrator responsible for deploying pods to a cluster and 
+maintaining the cluster.
+
+### Workload Provider
+This persona designs and creates the orchestration objects comprising the solution (e.g. 
+kubernetes pod descriptions etc). These objects reference containers published by Container Image Providers.
+In some cases the Workload and Container Image Providers may be the same entity.
+The solution defined is intended to provide the Application or Workload which in turn provides 
+value to the data owners (customers and clients).
+The workload provider and data owner could be part of same company/organisation but 
+following the least privilege principle the workload provider should not be able to view or 
+manipulate end user data without informed consent.
+- Need to prove to customer aspects of compliance.
+- Defines what the solution requires in order to run and maintain compliance (resources, utility 
+  containers/services,  storage).
```

Comment:
is it the container image provider who defines/locks the image config (e.g., cmd/entrypoint) that is only allowed to be used during runtime?

Reply:
Mm interesting question,
I guess for me the Workload Provider needs to which containers to use and in theory should be testing the solution out. So they will be defining what can run and how it can run including any image config.

To be honest I think this would be something to think about further, what can a Container image provider do and a workload provider chooses to inherit v what a workload provider can do regardless of the Container Image provider.
From a trust perspective the workload provider needs have the final say, but the mechanics of how to do say may need work from container image providers which the workload provider chooses to trust.

What thinking already exists within a kubernetes context today to lock the config such as cmd/entrypoint? Happy to be pointed to educate myself better.

Reply:
> To be honest I think this would be something to think about further, what can a Container image provider do and a workload provider chooses to inherit v what a workload provider can do regardless of the Container Image provider. From a trust perspective the workload provider needs have the final say, but the mechanics of how to do say may need work from container image providers which the workload provider chooses to trust.

Hmm... from a compliance perspective, I would also add the data owner into the mix. The data owner need to assure all regulatory requirements when using the data. So does it inherit what the persona gets from workload/container image owner (need for measurements?).